decidim-admin/app/helpers/decidim/admin/icon_link_helper.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    module IconLinkHelper
+      # This helper adds the necessary boilerplate for the admin icon links.
+      # 
+      # icon_name - A String representing the icon name from Iconic
+      #             http://useiconic.com/open
+      # link      - The path or url where the link should point to.
+      # title     - A String that will be shown when hovering the icon.
+      # options   - An optional Hash containing extra data for the link:
+      #             method - Symbol of HTTP verb. Supported verbs are :post, :get, :delete, :patch, and :put.
+      #             class  - Any extra class that will be added to the link.
+      #             data   - This option can be used to add custom data attributes.
+      #
+      def icon_link_to(icon_name, link, title, options = {})
+        link_to(link,
+                method: options[:method],
```

Comment:
I'd default `:method` to `:get`, just in case, as it's the default value for the `link_to` method.

`method: options[:method] || :get`